app-emulation/nesalizer/nesalizer-1.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="SDL-based NES emulator"
+DESCRIPTION="A work-in-progress NES emulator with a real-time rewind \
+feature that correctly reverses sound."
+HOMEPAGE="https://github.com/ulfalizer/nesalizer"
+COPYRIGHT="2013-2015 Ulf Magnusson"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="5bb045845a5cc829a78b7384f848fdd886cd98c8"
+SOURCE_URI="https://github.com/ulfalizer/nesalizer/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="4282cb0e4af0585af4a594dfa30b2e350dd0efc39e5bc2d8312f637f50397107"
+SOURCE_DIR="nesalizer-$srcGitRev"
+PATCHES="nesalizer-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	nesalizer$secondaryArchSuffix = $portVersion
+	app:Nesalizer
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libexecinfo$secondaryArchSuffix
+	devel:libsdl2_2.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	make CONF=release $jobArgs
+}
+
+INSTALL()
+{
+	make CONF=release INSTALL_DIR=$appsDir
+
+	addAppDeskbarSymlink $appsDir/Nesalizer "Nesalizer"
```

Comment:
I would put this in $prefix/bin ($binDir would resolve to ./self/bin/x86 on gcc2h but we don't want that if gcc2 version can't be build, also drop the symlink, it's a cli binary where you have to pass a <rom> file too.
```diff
-	make CONF=release INSTALL_DIR=$appsDir
-
-	addAppDeskbarSymlink $appsDir/Nesalizer "Nesalizer"
+	make CONF=release INSTALL_DIR=$prefix/bin
```

Reply:
The gcc2 version can't be built either way due to compiler incompatibilities, but I'm going to replace this anyways.

Reply:
Hence why we mark it as broken !x86_gcc2, thanks